tests/safety/test_toyota.py

```diff
@@ -316,6 +324,9 @@ class TestToyotaStockLongitudinalBase(TestToyotaSafetyBase):
   RELAY_MALFUNCTION_ADDRS = {0: (0x2E4,)}
   FWD_BLACKLISTED_ADDRS = {2: [0x2E4, 0x412, 0x191]}
 
+  def test_diagnostics(self, stock_longitudinal: bool = True):
```

Comment:
this argument default is confusing... is it used?

Reply:
yes, we use it to re-use the test from the longitudinal class we inherit from, both use it since there's conditional sending when stock ACC for canceling with ACC_CONTROL

a class variable probably might make more sense though